helm/cloud-provider-vsphere/values.yaml

```diff
@@ -1,62 +1,75 @@
-global:
-  config:
-    enabled: true
-    clusterId: ""
-    vcenter: ""
-    username: ""
-    password: ""
-    datacenter: ""
-    region: ""
-    zone: ""
-    # openssl s_client -connect <vcenter>:443 < /dev/null 2>/dev/null | openssl x509 -fingerprint -sha1 -noout
-    thumbprint: ""
+# Default values for vSphere CPI.
+# This is a YAML-formatted file.
+# vSphere CPI values are grouped by component
 
-  podSecurityStandards:
-    enforced: false
-
-cloud-provider-for-vsphere:
-  daemonset:
-    image: gsoci.azurecr.io/giantswarm/cloud-provider-vsphere
-
-kube-vip:
-  enabled: true
-  env:
-    vip_interface: "ens192"
-  image:
-    repository: gsoci.azurecr.io/giantswarm/kube-vip
-  nameOverride: kube-vip-svc-lb
-  tolerations:
-  - effect: NoSchedule
-    key: node-role.kubernetes.io/control-plane
-    operator: Exists
-  - effect: NoSchedule
-    key: node-role.kubernetes.io/master
-    operator: Exists
-
-kube-vip-cloud-provider:
+config:
```

Comment:
Is this going to work from the cluster chart? I see it still references `global` in your PR.
https://github.com/giantswarm/cluster-vsphere/pull/346/files#diff-bc0ebfac1325c3b172e0dc05668cc4ac64895f77aa6c78f1bbfe56d4fcd5e4caL45

Reply:
Yes this one caught me out initially too. But if you check `helm/cloud-provider-vsphere/charts/cloud-provider-for-vsphere/templates/secret.yaml` in the current chart on line 4 you can see that it accepts the values being passed under either `.config` or `.global.config` (I'm on my phone so I can't link directly). 

I elected to leave it under global in cluster-vsphere to minimise changes to that chart, but that may also be a bit unnecessary. I'm on the fence about it to be honest.

Reply:
I ended up changing it here https://github.com/giantswarm/cluster-vsphere/pull/346/commits/8977e872ccf80702a1e5afd94cf0dac5daeb2547